### ISSUE TITLE
Anything with type Any should be a valid base class

### ISSUE THREAD

**Issue by JukkaL** [JukkaL/mypy#940]
This should work:

```python
import m # type: ignore

class A(m.B):  # Currently complains about invalid type m.B
    def f(self): ...

a = A()
a.anything()  # OK
a.f(1)  # Error, f takes no arguments
```

If any base class has type `Any`, we shouldn't complain about missing attributes, but we can still give real types for explicitly defined methods and attributes.

**Comment by ecprice** [comment]
This is a dup of #864.
